src/main/java/io/fabric8/elasticsearch/plugin/acl/DynamicACLFilter.java

```diff
@@ -128,6 +129,8 @@ public void process(RestRequest request, RestChannel channel, RestFilterChain ch
 				if(updateCache(user, token, isClusterAdmin)){
 					syncAcl();
 				}
+
+				setDashboards(user, listProjectsFor(token), roles, esClient, kibanaIndex, kibanaVersion);
```

Comment:
move this so we don't have to make a call to get projects twice; we shouldn't be updating the dashboards unless we update the cache anyways